### PR TITLE
Added Image Input Condition

### PR DESCRIPTION
**Pull request by NitinBhasneria**
This PR is for repairing the image module for working fine with plots2.

### CODE REVIEW

**Comment by NitinBhasneria** [comment]
@jywarren have a look!

File: src/modules/PublicLab.MainImageModule.js [hunk 1]
```diff
@@ -147,27 +147,29 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
     var imageInput = document.getElementById('thumbnail-img');
     var infoArea = document.getElementById('thumbnail-filename');
 
-    imageInput.addEventListener('change', showFileName);
+    if (imageInput) {
```
Comment (sagarpreet-chadha) [inline src/modules/PublicLab.MainImageModule.js:150]: We should use `imageInput && infoArea` in if condition

**Comment by sagarpreet-chadha** [comment]
Let' merge this also:
https://github.com/publiclab/PublicLab.Editor/pull/588

File: src/modules/PublicLab.MainImageModule.js (hunk 1 above)
Comment (NitinBhasneria) [reply src/modules/PublicLab.MainImageModule.js:150]: if we use `image input` in if condition then we cant check it. By the way its working fine now,

Comment (sagarpreet-chadha) [reply src/modules/PublicLab.MainImageModule.js:150]: Yes good to know it is working 👍
I am recommending this change: 
```suggestion
    if (imageInput && infoArea) {
```

Let's say `thumbnail-filename` id is not present, then using `infoArea` variable will break UI, right?

Comment (NitinBhasneria) [reply src/modules/PublicLab.MainImageModule.js:150]: oh ok I got it.

**Comment by jywarren** [comment]
Great work, thanks folks!

**Pull request merged**